pycoverage.el

```diff
@@ -11,6 +11,10 @@
 
 ;; Thanks to bertrand.lallau@gmail.com for refactor
 
+;;; Code:
+
+(require 'cl-lib)
```

Comment:
Loading cl.el at compile time causes byte-compile warnings. cl.el functions and macros are alias of cl-lib.el since Emacs 24.3. And Emacs recommends developer to use cl-lib.el as below.

```
 If you need Common Lisp extensions, use the `cl-lib' library
     rather than the old `cl' library.  The latter does not use a clean
     namespace (i.e., its definitions do not start with a `cl-'
     prefix).  If your package loads `cl' at run time, that could cause
     name clashes for users who don't use that package.
```